README.md

```diff
@@ -45,7 +51,7 @@ $(document).ready(function () {
 
 ### Synchronous content initialization
 
-The following example can be used to set the `lhs` and `rhs` editors synchronously (i.e. their contents are already known):
+The following example can be used to set the `lhs` and `rhs` editors synchronously, that is, when their contents are already known):
```

Comment:
You removed the opening parenthesis, so the closing one is unbalanced. It should be restored:

```suggestion
The following example can be used to set the `lhs` and `rhs` editors synchronously (that is, when their contents are already known):
```